disttask/framework/storage/table_test.go

```diff
@@ -224,11 +224,19 @@ func TestSubTaskTable(t *testing.T) {
 	subtasks, err := sm.GetSucceedSubtasksByStep(2, proto.StepInit)
 	require.NoError(t, err)
 	require.Len(t, subtasks, 0)
+	subtaskMetas, err := sm.GetSucceedSubtaskMeta(2)
```

Comment:
Can you use a variable `taskIDxxx` to represent 2

Reply:
Then we need to change the whole file and bring unnecessary LOC... I prefer to do this in another PR.